hs/src/Reach/EditorInfo.hs

```diff
@@ -180,4 +188,4 @@ completionKind v =
         SLForm_apiCall_partial _ -> Nothing
         SLForm_wait -> Just CK_Function
         SLForm_setApiDetails -> Nothing
-    _ -> Nothing
+    --_ -> Nothing
```

Comment:
Delete, don't comment

Reply:
Oops, I put those comments while I was working on it and then forgot about them.